zer0-os/zOS
# feat(conversation-list-labels): update labels ui

NOTE :: I have removed 'archived' label and functionality from this PR to do in a follow up.

### What does this do?
- updates labels ui

### Why are we making this change?
- to add labels (expand from only 'All' or 'Favorites')

### How do I test this?
- run tests as usual.
- run ui, check conversation list

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?


https://github.com/user-attachments/assets/c4165c1d-7499-449b-ac4e-e0f1c319bd11
